runtime/compiler/control/CompilationThread.cpp

```diff
@@ -3116,8 +3116,13 @@ void TR::CompilationInfo::stopCompilationThreads()
 
       fprintf(stderr, "Number of compilations per level:\n");
       for (int i = 0; i < (int)numHotnessLevels; i++)
+         {	      
          if (_statsOptLevels[i] > 0)
-            fprintf(stderr, "Level=%d\tnumComp=%d\n", i, _statsOptLevels[i]);
+            fprintf(stderr, "Level=%d\tnumComp=%d\tnumRemoteComp=%d\n", i, _statsOptLevels[i], _statsRemoteOptLevels[i]);
+         }
+#if defined(J9VM_OPT_JITSERVER)
+      JITServerHelpers::printJITServerMsgStats(jitConfig);
+#endif /* defined(J9VM_OPT_JITSERVER) */
```

Comment:
You don't have to call `JITServerHelpers::printJITServerMsgStats(jitConfig);` because it's already called in jitShutdown where it prints stats about messages